<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollBar" name="horizontalScrollBarSugarLevel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelSugarLevel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sugar Level</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderBloodPressure">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBarPulseRate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderCholestrolLevel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelBloodPressure">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Blood Pressure</string>
   </property>
   <property name="buddy">
    <cstring>horizontalSliderBloodPressure</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelPulseRate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulse Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCholestrolLevel">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cholestrol Level</string>
   </property>
   <property name="buddy">
    <cstring>verticalSliderCholestrolLevel</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditResult">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalScrollBarSugarLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>scrollhorizontal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBloodPressure</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>sliderhorizontal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderCholestrolLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>slidervertical()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBarPulseRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>scrollvertical()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scrollhorizontal()</slot>
  <slot>scrollvertical()</slot>
  <slot>sliderhorizontal()</slot>
  <slot>slidervertical()</slot>
 </slots>
</ui>
